math/mathcore/test/stressGoFTest.cxx

```diff
@@ -555,7 +551,7 @@ Int_t stressGoFTest(Int_t argc = 1 , Char_t* argv[] = 0) {
    return RunTests(argc, argv);
 }
 
-#if !defined(__CINT__) && !defined(__MAKECINT__)
+#if !defined(__MAKECINT__)
```

Comment:
```suggestion
#if !defined(__CLING__)
```
This looks like the intent is to hide the main function from the interpreter.

Reply:
Oh, I accidentally uploaded these changes to the `test` folder. I think here the intent was to hide `main` so that the test could be run in the interpreter.

Reply:
Done.